source/manual/architecture-shallow-dive.html.md

```diff
@@ -0,0 +1,145 @@
+
+GOV.UK Origin is a stack of computers in the AWS cloud, and its entry point is a load balancer. Origin routes requests to the machine running the application that can respond to the request.
+
+Fastly also provides edge nodes or servers that are closer to our users, particularly those outside the UK.
+
+If Fastly has the homepage cached, Fastly returns the homepage to the user.
+
+If Fastly does not have the homepage cached, Fastly sends the request on to Origin. Origin is the infrastructure that GOV.UK maintains, hosted in Amazon Web Services (AWS).
+
+Within Origin, the request goes to an application called Router to determine which GOV.UK application will produce the page.
+
+The request from Router is then proxied (or forwarded) to an application called Frontend.
+
+[Frontend](https://github.com/alphagov/frontend) is a public-facing app that serves the majority of documents on the /government part of GOV.UK. These government pages are fetched from the Content Store by the appropriate application. Frontend is a replacement for the public-facing parts of the Whitehall application.
+
+Frontend produces the homepage as an HTML response and sends that response back to Router.
+
+Router sends the response to Fastly, which caches the response for 30 minutes. This means that Fastly can handle any subsequent requests for the homepage during this time, without needing to query Origin at all.
+
+Finally, Fastly sends the response back to the user.
+
+### A user visits a GOV.UK webpage
+
+The user searches for a GOV.UK webpage, and selects the webpage link in the search results. For example, this [news page about changes to travel between Nigeria and England](https://www.gov.uk/government/news/uk-changes-travel-rules-for-travellers-from-nigeria-to-england).
+
+The user's browser uses [DNS](#dns) to figure out where to send the request, and sends that request to the Fastly content delivery network (CDN).
+
+If Fastly has the homepage cached, Fastly returns the homepage to the user.
+
+If Fastly does not have the homepage cached, Fastly sends the request on to Origin. Origin is the infrastructure that GOV.UK maintains, hosted in Amazon Web Services (AWS).
+
+Within Origin, the request goes to an application called Router to determine which GOV.UK application will produce the page.
+
+Router gets the request and decides which application should render the page. Router then proxies (forwards) the request to the Frontend application.
+
+Frontend looks at the request and decides which content to get from the Content Store. The [Content Store](https://github.com/alphagov/content-store) is:
+
+- an application that exposes the [Content Store API](https://github.com/alphagov/content-store/blob/main/docs/content-store-api.md)
+- a MongoDB database of almost all the content published on GOV.UK, excluding dynamic elements such as top links on taxon pages, navigation elements, or search result pages
```

Comment:
I'd flip these around, along the lines of:

- a MongoDB database of almost all content published on GOV.UK...
- ...and an application that exposes this content via the Content Store API